### PR TITLE
Updated help guide

### PR DESCRIPTION
**Pull request by Preshita01**

### CODE REVIEW

**Review by ErnestCuong — APPROVED** [review]
LGTM

File: src/main/java/tutoraid/commons/core/HelpGuide.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
     public static final String ADD_L_TITLE = "Adding a lesson";
     public static final String ADD_L_DESC = "Adds a new lesson to TutorAid.";
-    public static final String ADD_L_COMMAND = "add -l s/LESSON_NAME [c/LESSON_PHONE] [p/LESSON_PRICE] "
+    public static final String ADD_L_COMMAND = "add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] "
             + "[t/LESSON_TIMING]";
     public static final String ADD_L_EXAMPLE = "add -l s/Maths 1 c/50 p/100 t/Mon 1000-1200";
```
Comment (JunWei3112) [inline src/main/java/tutoraid/commons/core/HelpGuide.java:47]: Need to change to `add -l n/Maths 1 c/50 p/100 t/Mon 1000-1200`

**Review by JunWei3112 — CHANGES_REQUESTED** [review]
One minor change to make

**Review by JunWei3112 — APPROVED** [review]
LGTM

**Pull request merged**